src/commonTest/kotlin/fr/acinq/lightning/serialization/StateSerializationTestsCommon.kt

```diff
@@ -1,69 +1,67 @@
 package fr.acinq.lightning.serialization
 
-import fr.acinq.bitcoin.Block
 import fr.acinq.lightning.Feature
 import fr.acinq.lightning.Lightning.randomKey
 import fr.acinq.lightning.MilliSatoshi
 import fr.acinq.lightning.channel.*
-import fr.acinq.lightning.tests.TestConstants
+import fr.acinq.lightning.serialization.Encryption.from
 import fr.acinq.lightning.tests.utils.LightningTestSuite
 import fr.acinq.lightning.wire.CommitSig
+import fr.acinq.lightning.wire.EncryptedChannelData
 import fr.acinq.lightning.wire.LightningMessage
 import fr.acinq.secp256k1.Hex
 import kotlin.math.max
-import kotlin.test.Test
-import kotlin.test.assertEquals
-import kotlin.test.assertFails
-import kotlin.test.assertTrue
+import kotlin.test.*
 
 class StateSerializationTestsCommon : LightningTestSuite() {
 
     @Test
     fun `serialize normal state`() {
         val (alice, bob) = TestsHelper.reachNormal()
-        val bytes = Serialization.serialize(alice)
-        val check = Serialization.deserialize(bytes, alice.staticParams.nodeParams)
-        assertEquals(alice, check)
+        val bytes = Serialization.serialize(alice.state)
+        val check = Serialization.deserialize(bytes)
+        assertEquals(alice.state, check)
 
-        val bytes1 = Serialization.serialize(bob)
-        val check1 = Serialization.deserialize(bytes1, bob.staticParams.nodeParams)
-        assertEquals(bob, check1)
+        val bytes1 = Serialization.serialize(bob.state)
+        val check1 = Serialization.deserialize(bytes1)
+        assertEquals(bob.state, check1)
     }
 
     @Test
     fun `encrypt - decrypt normal state`() {
         val (alice, bob) = TestsHelper.reachNormal()
-        val priv = randomKey().value
-        val bytes = Serialization.encrypt(priv, alice)
-        val check = Serialization.decrypt(priv, bytes, alice.staticParams.nodeParams)
-        assertEquals(alice, check)
-
-        val bytes1 = Serialization.encrypt(priv, bob)
-        val check1 = Serialization.decrypt(priv, bytes1, bob.staticParams.nodeParams)
-        assertEquals(bob, check1)
+        val priv = randomKey()
+        val bytes = EncryptedChannelData.from(priv, alice.state)
+        val check = ChannelStateWithCommitments.from(priv, bytes)
+        assertEquals(alice.state, check)
+
+        val bytes1 = EncryptedChannelData.from(priv, bob.state)
+        val check1 = ChannelStateWithCommitments.from(priv, bytes1)
+        assertEquals(bob.state, check1)
     }
-
+    
+    @Ignore
```

Comment:
We can probably completely remove that test?

Reply:
Done in c5365f4.